Tests.Integration.Presentation.Web/KLE/V2/KleOptionV2ApiTest.cs

```diff
@@ -121,8 +120,8 @@ public async Task Can_Get_Kle_Numbers_By_Description_Content()
             var token = await HttpApi.GetTokenAsync(OrganizationRole.User);
             var content = A<string>();
             PurgePreviousTestData();
-            var match1 = CreateTaskRefInDb(100, 10, 100, description: $"{content}{A<string>()}");
-            var match2 = CreateTaskRefInDb(100, 15, 100, description: $"{A<string>()}{content}{A<string>()}");
+            var match1 = CreateTaskRefInDb(100, 10, 100, description: CreateDescription(content));
+            var match2 = CreateTaskRefInDb(100, 15, 100, description: $"{A<string>()}{CreateDescription(content)}");
```

Comment:
you are not using the danish chars because it's the "content" that is being passed as query string